src/main.js

```diff
@@ -23,6 +23,9 @@ app.use(store);
 // Add localization to vue
 app.use(i18n);
 
+//Using Language from Loacal Storage if not found than by defualt English is selceted
+i18n.global.locale.value = getLocalStorage('preferredLocale') ?? 'en';
```

Comment:
I'm not sure this is the right way to do it. Would it be better to set the default initially when loading i18n in the first place instead of switching after the app initializes?

Reply:
like before app uses the i18n right.

Reply:
More like in here, https://github.com/cuttle-cards/cuttle/blob/main/src/i18n.js, as we're instantiating i18n.

We will also need to validate that it's a valid locale in case anyone changes the value. Maybe validate against the files that exist in the locale directory. @itsalaidbacklife, Penny for your thoughts.